jsonschema-core.xml

```diff
@@ -2872,73 +2896,174 @@ https://example.com/schemas/common#/$defs/count/minimum
                             failure
                         </t>
                         <t>
-                            "nested" - the collection of errors or annotations produced by a keyword
+                            "nested" - the collection of errors or annotations produced by a subschema
```

Comment:
Here I would use "schema object", because root schemas also produce output units.

Reply:
"Subschema" is proper here.  The context is the `hierarchical` output.  In this case, `nested` holds output units from strictly subschemas.